src/main/java/org/immregistries/smm/transform/procedure/AlternativeEndingVowels.java

```diff
@@ -121,6 +121,9 @@ protected static String varyText(String name, Transformer transformer, Field fie
         case LAST_NAME:
         case MOTHERS_MAIDEN_NAME:
         case ADDRESS_STREET:
+          name = ALTERNATIVE_LAST_NAMES.get(random.nextInt(ALTERNATIVE_LAST_NAMES.size()));

```

Comment:
I don't quite get why this was added. I think it's okay. Assuming that this was a partial thought that there might be a difference but then that wasn't the direction you went.

Reply:
Hmm, yeah I think the lost thought was to make the email address valid in the switch statement, but then I realized that the rest of the email was built down below.